src/listeners/reactionPin.ts

```diff
@@ -0,0 +1,41 @@
+import {Events, Listener, PieceContext} from '@sapphire/framework';
+import {getSettings} from '../database/models/SettingsModel';
+import colors from '../utils/colorUtil';
+import {BediEmbed} from '../lib/BediEmbed';
+import {MessageReaction, User} from 'discord.js';
+import logger from '../utils/loggerUtil';
+
+module.exports = class PinReactionListener extends Listener {
+  constructor(context: PieceContext) {
+    super(context, {
+      event: Events.MessageReactionAdd,
+    });
```

Comment:
Not sure how the listener stuff works, but we should verify that we have manage channel perms